Stop using short open tags (resolves VUFIND-1257).

demiankatz opened the pull request:
TODO
- [x] Discuss whether this is the best option, and whether we should clean up any formatting
- [x] Add note to changelog

demiankatz commented:
I am very puzzled by the linting failures that Travis is reporting across all PHP versions. I cannot reproduce these problems on my local server. Brilliant ideas welcomed.

xmorave2 commented on themes/bootstrap3/templates/Auth/MultiILS/loginfields.phtml:
```diff
@@ -4,7 +4,7 @@
   <select id="login_target" name="target" class="form-control">
     <?foreach ($this->auth()->getManager()->getLoginTargets() as $target):?>
```
Demian, could be this the problem? The opening tag is not followed by space and then not catched by this patch.

xmorave2 replied:
I just looked into the first file reported by Travis, the other files could have the same or other problem ;)

EreMaijala commented:
@demiankatz Travis php has short_open_tag disabled, and disabling them locally allowed me to also get the same results as Travis did.

demiankatz commented:
Good eyes, @xmorave2 and @EreMaijala. That makes perfect sense. We're closed today due to winter weather, but I'll get this sorted out when I return. Thanks!

EreMaijala commented:
I pushed a change to increase the verbosity of errors from php-cs-fixer. Maybe it'll help diagnose any issues in the future.

demiankatz commented:
@EreMaijala, thanks for the verbosity increase, that will be helpful in future -- I've moved it over to master. I've also fixed the missing spaces @xmorave2 noticed in master, and rebuilt this branch based on that cleaned-up code. I think we're in much better shape now. Some of the templates are a bit uglier with the full php tags, but I think we'll have to live with that for the benefits it provides us. Let me know if you have any further thoughts on this -- we'll likely discuss it in more detail on next week's developers call.
